dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -26,9 +26,9 @@ def replace_hostname(url)
   @browser.find_element(selection_criteria)
 end
 
-When /^I wait until element "([^"]*)" has text "([^"]*)"$/ do |selector, text|
+When /^I wait until (?:element )?"([^"]*)" (?:has|contains) text "([^"]*)"$/ do |selector, text|
   wait = Selenium::WebDriver::Wait.new(:timeout => 60 * 2)
-  wait.until { element_has_text(selector, text) }
+  wait.until { @browser.execute_script("return $(\"#{selector}\").text();").include? text }
 end
```

Comment:
This step definition was not used anywhere, and was actually broken before (`element_has_text` contained an assertion so it would fail the test the first time it was checked).